src/plugins/tables/tables-fltrs-en.hbs

```diff
@@ -0,0 +1,292 @@
+---
+{
+	"title": "Tables Filtering Example",
+	"language": "en",
+	"category": "Plugins",
+	"description": "Integrates the DataTables plugin into WET providing searching, sorting, filtering, pagination and other advanced features for tables.",
+	"tag": "tables filtering example",
+	"parentdir": "tables filtering example",
+	"altLangPrefix": "tables filtering example",
+	"dateModified": "2020-10-22"
+}
+---
+<section>
```

Comment:
Can you add a section that lists what filtering option is available, and include notes for author as needed on how to use that options.